flink-core/src/main/java/org/apache/flink/configuration/ResourceManagerOptions.java

```diff
@@ -56,14 +56,14 @@
 			" Its not possible to use this configuration key to define port ranges.");
 
 	/**
-	 * Percentage of heap space to remove from containers (YARN / Mesos), to compensate
+	 * Percentage of heap space to remove from containers (YARN / Mesos/ Kubernetes), to compensate
 	 * for other JVM memory usage.
 	 */
 	public static final ConfigOption<Float> CONTAINERIZED_HEAP_CUTOFF_RATIO = ConfigOptions
```

Comment:
My previous thought for this option was for all the containers including `TaskExecutor`. But after reviewing the related codes I find it is only for master container. The `TaskExecutor` container would never use it.

So maybe we can rename this option to limit the scope to make it more clearly, but it is out of scope of this PR.

Reply:
Yes, you are right. Currently, the cutoff is only effective for jobmanager. And i think we do need to rename it, because it will be deprecated soon. You could the discussion under https://issues.apache.org/jira/browse/FLINK-15288.